osu.Game/Rulesets/Mods/IMod.cs

```diff
@@ -11,7 +11,22 @@ public interface IMod : IEquatable<IMod>
         /// <summary>
         /// The shortened name of this mod.
         /// </summary>
-        [JsonProperty("acronym")]
         string Acronym { get; }
+
+        /// <summary>
+        /// The icon of this mod.
+        /// </summary>
+        IconUsage? Icon { get; }
+
+        /// <summary>
+        /// Whether this mod is playable by an end user.
+        /// Should be <c>false</c> for cases where the user is not interacting with the game (so it can be excluded from multiplayer selection, for example).
+        /// </summary>
+        bool UserPlayable { get; }
+
+        /// <summary>
+        /// Create a fresh <see cref="Mod"/> instance based on this mod.
+        /// </summary>
+        Mod CreateInstance() => ((Mod)this).DeepClone();
```

Comment:
This should probably use `Activator.CreateInstance()`, because it's a clone and not a "fresh" instance as the xmldoc implies.

Reply:
Yeah, fair point.